packages/sftp-client/index.js

```diff
@@ -105,7 +105,8 @@ class SftpClient {
    * @param {string} remotePath - the full path to the remote file to be fetched
    * @param {string} bucket - destination s3 bucket of the file
    * @param {string} key - destination s3 key of the file
-   * @returns {Promise} s3 uri of destination file
+   * @returns {Promise.<{ s3uri, etag }>} promise resolving to an object
```

Comment:
I don't know jsdoc well enough, is there a way to indicate that s3uri and etag are strings?

Reply:
I noticed that I failed to be consistent because there is one place where I do this, but not sure if it is JSDoc-friendly:
```
@returns {Promise.<{ s3uri: string, etag: string }>}
```